lsp-mode.el

```diff
@@ -3200,7 +3200,9 @@ disappearing, unset all the variables related to it."
                                                                                                   "refactor.inline"
                                                                                                   "refactor.rewrite"
                                                                                                   "source"
-                                                                                                  "source.organizeImports"])))))))
+                                                                                                  "source.organizeImports"])))))
+                                     (resolveSupport . ((properties . ["edit" "command" "kind"])))
```

Comment:
hm, we perform actions like filtering based on "kind". In that case, it seems to me that it is better to always request kind. WDYT?

Reply:
Hum, I think it makes sense, I'll remove it from here